packages/reporter/src/commands/command.jsx

```diff
@@ -118,13 +127,37 @@ class Command extends Component {
               <Tooltip placement='top' title={`${model.numElements} matched elements`}>
                 <span className='num-elements'>{model.numElements}</span>
               </Tooltip>
+              <Tooltip placement='top' title={`This event occurred ${model.numDuplicates} times`}>
```

Comment:
I would hide the number if `command-is-open`, it looks weird when it's expanded and still showing '5' events on the first one...like, almost looks like there's 10 events??

Reply:
Good idea.